packages/@expo/cli/src/start/server/metro/instantiateMetro.ts

```diff
@@ -105,7 +105,6 @@ export async function loadMetroConfigAsync(
     routerDirectory: getRouterDirectoryModuleIdWithManifest(projectRoot, exp),
     config,
     platformBundlers,
-    isTsconfigPathsEnabled: !!exp.experiments?.tsconfigPaths,
```

Comment:
Leave this in so users can toggle it off in case there's some case we don't cover.